examples/nextjs-spa/src/pages/recovery.tsx

```diff
@@ -87,17 +69,15 @@ const Recovery: NextPageWithLayout = () => {
 
     // If ?flow=.. was in the URL, we fetch it
     if (flowId) {
-      getRecoveryFlow(flowId).catch(
-        (err: AxiosError) =>
-          err.response?.status === 410 ?? createRecoveryFlow(returnTo),
-      )
+      getFlow(flowId).catch(() => createFlow(returnTo))
       // if the flow is expired, we create a new one
       return
     }
 
     // Otherwise we initialize it
-    createRecoveryFlow(returnTo)
-  }, [flowId, returnTo, getRecoveryFlow, createRecoveryFlow, router.isReady])
+    createFlow(returnTo)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm not sure if we should disable linter rules like this.
Since the functions are memoized through the `useCallback` function, it should be alright to add them to this dependency list.